docs/security/fields-access-control.md

```diff
@@ -6,6 +6,12 @@ An access control can be added on each field using `config.fields.*.access` or g
 If `config.fields.*.access` value is true field will be normally resolved but will be `null` otherwise.
 Act like access is`true` if not set.
```

Comment:
```suggestion
If `config.fields.*.access` value is true, field will be resolved normally, otherwise will be `null`.
Acts like `access` is`true` if not set.
```